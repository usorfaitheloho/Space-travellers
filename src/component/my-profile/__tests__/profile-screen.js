import React from 'react';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';
import store from '../../../redux/configureStore';
import MyProfile from '../profile';

test('My Profile renders correctly', () => {
  const tree = renderer.create(
    <Provider store={store}>
      <MyProfile />
    </Provider>,
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
