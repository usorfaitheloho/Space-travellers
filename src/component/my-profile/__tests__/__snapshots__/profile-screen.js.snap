// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My Profile renders correctly 1`] = `
<div
  className="profile"
>
  <section
    className="my-missions-sec"
  >
    <h3
      className="my-heading"
    >
       My Missions
    </h3>
    <div
      className="my-missions"
    >
      <h2>
        No Reserved Missions
      </h2>
    </div>
  </section>
  <section
    className="my-rockets-sec"
  >
    <h3
      className="my-heading"
    >
       My Rockets 
    </h3>
    <div
      className="my-rockets"
    >
      <h2>
        No Reserved Rockets
      </h2>
    </div>
  </section>
  <section
    className="my-dragons-sec"
  >
    <h3
      className="my-heading"
    >
      My Dragons
    </h3>
    <div
      className="my-dragons"
    >
      <h2>
         No reserved Dragons
      </h2>
    </div>
  </section>
</div>
`;
